packages/web-components/src/components/textarea/textarea.mdx

```diff
@@ -1,76 +1,52 @@
-import { ArgTypes, Markdown, Meta } from '@storybook/blocks';
-import { cdnJs, cdnCss } from '../../globals/internal/storybook-cdn';
-import * as TextArea from './textarea.stories';
+import { ArgTypes, Canvas, Markdown, Meta } from '@storybook/blocks';
+import { cdnJs } from '../../globals/internal/storybook-cdn';
+import * as TextAreaStories from './textarea.stories';
 
-<Meta of={TextArea} />
+<Meta of={TextAreaStories} />
 
-# Textarea
+# Text Area
 
-> 💡 Check our
-> [Stackblitz](https://stackblitz.com/github/carbon-design-system/carbon/tree/main/packages/web-components/examples/components/textarea)
-> example implementation.
+[Source code](https://github.com/carbon-design-system/carbon/tree/main/packages/web-components/src/components/text-area)
+&nbsp;|&nbsp;
+[Usage guidelines](https://www.carbondesignsystem.com/components/text-area/usage)
+&nbsp;|&nbsp;
+[Accessibility](https://www.carbondesignsystem.com/components/text-area/accessibility)
```

Comment:
```suggestion
```

It looks like `textarea` isn't listed in the components section on the Carbon website. @laurenmrice @aagonzales Is this intentional?

If so Ari, we should remove the breadcrumb links from the React docs as well.

Reply:
Oh yeah `text area` doesn't have its own page, its grouped in with `text input`: https://carbondesignsystem.com/components/text-input/usage/#text-area

Reply:
thanks updated!